File: src/main/java/no/nav/pensjon/innsyn/sink/CellValueSetter.java
package no.nav.pensjon.innsyn.sink;

import java.util.Date;

public interface CellValueSetter {

    void setCellValue(int cellIndex, boolean value);

    void setCellValue(int cellIndex, Date value);

    void setCellValue(int cellIndex, double value);

    void setCellValue(int cellIndex, String value);
}

File: src/main/kotlin/no/nav/pensjon/innsyn/domain/map/CellValueSetter.kt
package no.nav.pensjon.innsyn.domain.map

import java.util.*

interface CellValueSetter {
    fun setCellValue(cellIndex: Int, value: Boolean)
    fun setCellValue(cellIndex: Int, value: Date)
    fun setCellValue(cellIndex: Int, value: Double)
    fun setCellValue(cellIndex: Int, value: String)
}